profiles/netcoreall.props

```diff
@@ -0,0 +1,14 @@
+<Project>
+  <!-- All .NET Core / .NET Standard TFMs -->
+  <PropertyGroup>
+    <FakeItEasyTargetFrameworks>netstandard1.6;netstandard2.0;netstandard2.1</FakeItEasyTargetFrameworks>
+    <ValueTaskExtensionsTargetFrameworks>netstandard1.6;netstandard2.0;netstandard2.1</ValueTaskExtensionsTargetFrameworks>
+    <ExampleTargetFrameworks>netcoreapp2.1</ExampleTargetFrameworks>

```

Comment:
I expected 2.1 and 3.0.
If the thinking was that they're just examples, so we probably don't need to build both, maybe default to the newer framework? I can live with it as is, though.

(Actually, this brings up a similar question for the full profile, and which frameworks we should target for the examples and the approval tests - if we're just going to target one in each build profile, should it be the latest/most portable? Or maybe it's all fine.)

Reply:
> If the thinking was that they're just examples, so we probably don't need to build both

Yes, that's what I had in min

> maybe default to the newer framework?

👍 

> (Actually, this brings up a similar question for the full profile, and which frameworks we should target for the examples and the approval tests - if we're just going to target one in each build profile, should it be the latest/most portable? Or maybe it's all fine.)

The latest for each profile is a good choice, I think.